universal/ubi8/Dockerfile

```diff
@@ -63,10 +63,10 @@ ENV PATH="/home/user/.local/share/coursier/bin:$PATH"
 
 # NodeJS
 ENV NVM_DIR="/home/user/.nvm"
-ENV NODEJS_VERSION=16.14.0
```

Comment:
maybe it's better to keep having node 16 and remove node 12.
so, we'll have 18 (default), 16 and 14

Reply:
better than having node 20 (default), 18, and 16 (EOL) ?

Reply:
I'd set 18 (LTS) as a default.
@azatsarynnyy is it OK for che-code if UDI provides node 18 as a default version of node?

Reply:
since 16 is EOL why should we have and support it in our product images?

Reply:
cc @RomanNikitenko

`Empty workspace smoke test on udi8` is passing and it should verify that vs-code starts successfully

Reply:
> since 16 is EOL why should we have and support it in our product images?

💯 agree... and yet we have these old, EOL'd versions in the Che UDI image:

```
ENV NODEJS_12_VERSION=12.22.10
ENV NODEJS_14_VERSION=14.19.0
```

Why??

Reply:
> is it OK for che-code if UDI provides node 18 as a default version of node?

At the moment VS Code uses [Node.JS version >= 18.15.x and < 19](https://github.com/microsoft/vscode/wiki/How-to-Contribute#prerequisites)